File: Programmers/level1/sumMatrix/sum_matrix.py
# 문제 출처 : https://programmers.co.kr/learn/courses/30/lessons/12950

def solution(arr1, arr2):
    for i in range(len(arr1)):
        for j in range(len(arr1[0])):
            arr1[i][j] += arr2[i][j]

    print(arr1)
    return arr1

# zip 함수
# 참고 레퍼런스 : https://wikidocs.net/32#zip

# def solution(A, B):
#     return [[c + d for c, d in zip(a, b)] for a, b in zip(A, B)]

File: Programmers/level1/sumMatrix/sumMatrix.js
// https://programmers.co.kr/learn/courses/30/lessons/12950?language=javascript

export const solution = (arr1, arr2) => {
    let answer = [];
    for(let i = 0; i < arr1.length; i++){
        let arr = [];
        for(let j = 0; j < arr1[0].length; j++){
            arr.push(arr1[i][j]+arr2[i][j]);
        }
        answer.push(arr);
    }
    return answer;
};